.github/workflows/build.yml

```diff
@@ -30,6 +30,12 @@ jobs:
         env:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: JavaDoc
```

Comment:
This will be executed for all branches, possibly overwriting the doc with stuff from feature branches and pull requests. I would either restrict it to the master branch or better yet, wait for the #112 PR and only apply it on the version tags.